Sources/OpenAPIRuntime/Interface/CurrencyTypes.swift

```diff
@@ -47,20 +82,12 @@ extension HeaderField {
     /// Use this to avoid leaking sensitive tokens into application logs.
     public static var redactedHeaderFields: Set<String> {
```

Comment:
This is actually a bit surprising to me. This is a global static which can be set to redact header fields? This sounds to be like something that should be rather passed in a configuration struct. @czechboy0 @simonjbeaumont is this something you considered before?

Reply:
Got you. We had a similar discussion recently in AHC https://github.com/swift-server/async-http-client/issues/689.

I don't know if I share the safety concern of including everything in `CustomStringConvertible`. If somebody `print()` a request or response then they must expect that there are sensitive informations in there. This can not only be in the headers but also in the path.

Personally, I would lean towards including everything in the `description` and it is up to the user if they log a request or not. If they do log a request they are themselves responsible for making sure it doesn't include sensitive data as long as we don't log it ourselves I don't see us in the responsibility to sanitise those types. Happy to move this discussion to a different place though.

Reply:
I also don't love that we do this and see this as a bit of a layering violation and the filtering could easily be achieved in a logging middleware. 

My biggest concern is it implies "safe default", which is really best-effort at best.

Reply:
I appreciate these thoughts, but as @FranzBusch suggested, let's move this to a dedicated issue/thread. This PR is about passing stricter concurrency checking.

Reply:
Sure—not suggesting we derail this PR. I was mentioned in this thread for an opinion and there is no other issue currently, so felt rude not to respond when asked 😕

Reply:
Yeah not suggesting this was derailing it, just that this is a good discussion and we should preserve the points in a separate issue, ideally with someone suggesting what we should change it _to_, with arguments of how that's overall a better solution. 🙂

Reply:
Moved to https://github.com/apple/swift-openapi-generator/issues/113.